### PR TITLE
CircleCI

### PR DESCRIPTION
**Pull request by mobileoverlord**
See https://github.com/nerves-project/nerves_system_rpi0/pull/25 for more info

### CODE REVIEW

**Comment by mobileoverlord** [comment]
This is ready for review

**Comment by mobileoverlord** [comment]
I am working on exposing this info, but I just plucked this from the test farm
```
%NervesTestServer.Build{__meta__: #Ecto.Schema.Metadata<:loaded, "builds">,
  device: "nerves-b8bc", end_time: #Ecto.DateTime<2017-08-18 21:05:29>,
  id: "328bd98c-f0ff-45e0-89b4-aad154a6d08f",
  inserted_at: ~N[2017-08-18 21:04:45.962935], org: "nerves-project",
  result: %{"failures" => 0, "skipped" => 0, "total" => 3},
  result_io: "...\n\nFinished in 0.7 seconds\n3 tests, 0 failures\n\nRandomized with seed 281126\n",
  start_time: #Ecto.DateTime<2017-08-18 21:05:00>, system: "nerves_system_rpi3",
  updated_at: ~N[2017-08-18 21:05:29.766290],
  vcs_id: "5d5a534526031756bcfb639013941d2cafb2e397"}
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+          context: org-global
+          requires:
+            - build_test
+      - deploy_system:
+          context: org-global
+          requires:
+            - build_system
+          filters:
+            branches:
+              only: master
```
Comment (fhunleth) [inline .circleci/config.yml]: Are the `filters` options a logical AND or logical OR? I.e. will a release get deployed if it is tagged on a branch or does it have to be a tag off `master`?

Comment (mobileoverlord) [inline .circleci/config.yml]: From the Circle Docs, here is an example of multiple filters
```
workflows:
  version: 2
  build-n-deploy:
    jobs:
      - build:
          filters:
            tags:
              only: /.*/
      - deploy:
          requires:
            - test
          filters:
            tags:
              only: /^config-test.*/
            branches:
              ignore: /.*/
```
https://circleci.com/docs/2.0/workflows/#git-tag-job-execution

**Pull request closed without merging**